### PR TITLE
Multi groups

### PR DESCRIPTION
**Pull request by hanahmily**
Related to apache/skywalking#12134

### CODE REVIEW

**Comment by wu-sheng** [comment]
Oops, CI fails, please fix.

File: src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java [hunk 1]
```diff
@@ -348,11 +348,18 @@ public StreamWrite createStreamWrite(String group, String name, final String ele
...
-                        .query(streamQuery.build(this.metadataCache.findMetadata(streamQuery.group, streamQuery.name))));
-        return new StreamQueryResponse(response);
+        for (String group : streamQuery.groups) {
+            MetadataCache.EntityMetadata em = this.metadataCache.findMetadata(group, streamQuery.name);
+            if (em != null) {
+                final BanyandbStream.QueryResponse response = HandleExceptionsWith.callAndTranslateApiException(() ->
+                        this.streamServiceBlockingStub
+                                .withDeadlineAfter(this.getOptions().getDeadline(), TimeUnit.SECONDS)
+                                .query(streamQuery.build(em)));
+                return new StreamQueryResponse(response);
```
Comment (lujiajing1126) [inline src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java:358]: It seems only the first group is used

Comment (hanahmily) [reply src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java:358]: Sorry, it seems inappropriate to handle this. I will fix it.

Comment (hanahmily) [reply src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java:358]: No, streamQuery includes all groups. Here, I simply use the first group to determine the resource's tag family and build the query request. This operation assumes that resources within these groups have the same tag family and tags.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**